genestack_client/scripts/genestack_application_manager.py

```diff
@@ -204,8 +204,8 @@ def update_parser(self, p):
             help='Visibility level which will be set to application: %s' % (' | '.join(VISIBILITIES))
         )
         p.add_argument(
-            'accessions', metavar='<accessions>', nargs='*',
-            help='Accessions of groups for group visibility rule'
+            'accessions', metavar='<groups_accessions>', nargs='*',
+            help='Accessions of groups for \'group\' visibility rule'
```

Comment:
`'Accessions of groups for \'group\' visibility rule'` -> `"Accessions of groups for 'group' visibility rule"`